editor/editor_property_name_processor.cpp

```diff
@@ -236,6 +236,7 @@ EditorPropertyNameProcessor::EditorPropertyNameProcessor() {
 	capitalize_string_remaps["lod"] = "LOD";
 	capitalize_string_remaps["lods"] = "LODs";
 	capitalize_string_remaps["lowpass"] = "Low-pass";
+	capitalize_string_remaps["loongarch64"] = "loongarch64";
```

Comment:
These are ordered alphabetiically so `loongarch64` should come between `lods` and `lowpass`.